docs/libraries/nodejs/getting_started.md

```diff
@@ -2,17 +2,19 @@
 
 The [IOTA Wallet Node.js binding](https://www.npmjs.com/package/@iota/wallet) is published on npmjs.org.
 
+> There is a guide for exchanges [available](https://chrysalis.docs.iota.org/guides/exchange_guide.html) which is based on `wallet.rs` and `Node.js`. It also covers several most common use cases.
+
 ## Installation
 
-Currently the package isn't published so you'd need to link it to your project using `npm` or `yarn`.
+Currently the package isn't published so you'd need to link it to your project using `npm` or `yarn`. We also recommend to use `dotenv` for password management.
```

Comment:
@nothingismagick what do you recommend for stronghold password management?

Reply:
I added a general notice and added link to a security and backup recommendations on Chrysalis docs. But will be glad to add any additional specific recommendations by @nothingismagick here or also on Chrysalis Docs. Please, kindly let me know.